inbm/dispatcher-agent/tests/unit/source/test_ubuntu_source_cmd.py

```diff
@@ -227,7 +227,7 @@ def test_add_app_deb_822_format_successfully(self):
     def test_update_app_source_successfully(self):
         try:
             params = ApplicationUpdateSourceParameters(
-                file_name="intel-gpu-jammy.list", sources=APP_SOURCE
+                source_list_file_name="intel-gpu-jammy.list", sources=APP_SOURCE
             )
             command = UbuntuApplicationSourceManager()
             with patch("builtins.open", new_callable=mock_open()):
```

Comment:
**Suggestion:** Use a context manager to ensure that the file is properly closed after the operation is complete. [best practice]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
            with patch("builtins.open", new_callable=mock_open()) as mocked_file:
                command.add(params)
                mocked_file.assert_called_with(params.source_list_file_name, 'w')
```